### PR TITLE
Change/institutions

### PR DESCRIPTION
**Pull request by luisahorsten**

### CODE REVIEW

File: content/change/institutions/uni.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+# How can I start an Open Science Initiative at my own university?
+
+For starters, you might want to find out, if there is a national network of open-science initiatives in your country (e.g. [NOSI](https://osf.io/tbkzh/) in Germany, or [UK-OSWG](https://osf.io/vgt3x/) in the UK) and get in touch. They will most likely be happy to support you in your endeavour!
+
+Other than that, you can turn to these resources
+
+* The newly founded Open Science Community at the University Utrecht documented [why and how](https://docs.google.com/document/d/112B-OCArTI-zdDZ6S67BPVHSn6gPL9GgmluoQ35zG40/edit) they started their local Open Science Initiative:
+
+* There is also this guide on [how to establish an Open Science Interest Group]( https://docs.google.com/document/d/112B-OCArTI-zdDZ6S67BPVHSn6gPL9GgmluoQ35zG40/edit), but sadly we haven't been able track down the author of this piece. If you have any hints, please do let us know!
```
Comment (FelixHenninger) [inline content/change/institutions/uni.md]: This is the same link as above, is that intentional?

File: content/change/institutions/lab.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+** Or get in touch with one of the speakers on [this list](https://osf.io/tbkzh/wiki/Speakers%20for%20talks%20and%20workshops%20on%20Open%20Science%20topics/), who cover several topics in the field of Open Science.
+* If you're in one of those labs, where people are indifferent about replicability or even openly take a stand against it, you might find some arguments for your next discussion in one of these resources:
+** [Zwaan, R., Etz, A., Lucas, R. E., & Donnellan, B. (2018, August 1). Making Replication Mainstream.](https://doi.org/10.31234/osf.io/4tg9c)
+** [Open Science Talking Points](https://docs.google.com/document/d/1CJFdbxD9Z7xCWpXPW8XIKKpoXK-TGk9_SsVIqw9FhiM/edit), assembled at the SIPS Meeting 2018
+
+# What should I think of, when I want to transition my lab to open science practices?
+
+The best piece of advice is probably to take it step by step and don't put too much pressure on it. Research is a continuous process, so you can try to implement new practices along the way. (To make that a bit easier, we tried to structure this ressource based on the scientific circle.)
```
Comment (FelixHenninger) [inline content/change/institutions/lab.md]: > take it step by step and don't put too much pressure on it

I totally agree, but would tend to be careful with global statements like this. Would it be a compromise to just say "we think it's totally legitimate to move in small steps"

File: content/change/institutions/lab.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+* Some people at the SIPS Meeting 2018 put together some [links to useful resources](https://osf.io/f529n/wiki/home/ ) for transitioning to open science practices in labs.
```
Comment (FelixHenninger) [inline content/change/institutions/lab.md]: I think there are some great resources in this collection, but they look fairly generic to me -- I'm not sure they are particularly suited to this particular question.

File: content/change/institutions/uni.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+title: uni
+
+# How can I start an Open Science Initiative at my own university?
+
+For starters, you might want to find out, if there is a national network of open-science initiatives in your country (e.g. [NOSI](https://osf.io/tbkzh/) in Germany, or [UK-OSWG](https://osf.io/vgt3x/) in the UK) and get in touch. They will most likely be happy to support you in your endeavour!
```
Comment (FelixHenninger) [inline content/change/institutions/uni.md]: Could we be just a bit more careful about making promises for other people (about the supporting part).

File: content/change/institutions/lab.md (hunk 3 above)
Comment (FelixHenninger) [reply content/change/institutions/lab.md]: Can we use some of the links elsewhere?

File: content/change/institutions/uni.md (hunk 1 above)
Comment (luisahorsten) [reply content/change/institutions/uni.md]: Argh, no! Sorry and thanks for paying attention...

File: content/change/institutions/lab.md (hunk 2 above)
Comment (luisahorsten) [reply content/change/institutions/lab.md]: Sure!

File: content/change/institutions/uni.md (hunk 4 above)
Comment (luisahorsten) [reply content/change/institutions/uni.md]: You're absolutely right, sorry!

**Comment by FelixHenninger** [comment]
Ok, closing this for now.

**Pull request closed without merging**